pyfar/dsp/regularization.py

```diff
@@ -69,9 +69,10 @@ def from_frequency_range(cls, signal, frequency_range,
         Regularization from a given frequency range.
 
         Defines a frequency range within which the regularization
-        :math:`\epsilon(f)` is set to `regularization_within`.
+        :math:`\epsilon(f)` is set to ``regularization_within``.
```

Comment:
According to our guidelines it should indeed be in single quotes. We use italic font to reference parameters and monospaced font (double quotes) to indicate actual code. Can you change this here and below?